climt/_components/surface_ice.py

```diff
@@ -212,29 +180,29 @@ def __call__(self, state, timestep):
 
                     # Create temperature profile from cubic spline
                     vertical_coordinates = np.linspace(0, total_height, num_layers)
-                    temp_profile = input_arrays['snow_ice_temperature_poly'][lon, lat](vertical_coordinates)
+                    temp_profile = input_arrays['snow_and_ice_temperature_spline'][lon, lat](vertical_coordinates)
 
                     snow_level = int((1 - snow_height_fraction)*num_layers)
                     levels = np.arange(num_layers)
                     # print('snow_level', snow_level)
                     # print('num_layers', num_layers)
 
                     # Create vertically varying profiles
-                    rho_snow_ice = self._rho_ice*np.ones(num_layers)
-                    rho_snow_ice[levels > snow_level] = self._rho_snow
+                    rho_snow_ice = self._rho_ice.values.item()*np.ones(num_layers)
+                    rho_snow_ice[levels > snow_level] = self._rho_snow.values.item()
 
-                    heat_capacity_snow_ice = self._C_ice*np.ones(num_layers)
-                    heat_capacity_snow_ice[levels > snow_level] = self._C_snow
+                    heat_capacity_snow_ice = self._C_ice.values.item()*np.ones(num_layers)
```

Comment:
No change needed here, but maybe I should be putting in array.value as a shortcut to array.values.item() within the sympl.DataArray API.

Reply:
That will make the syntax much more natural. Let me know if you do this!

Reply:
Started discussion (here)[https://github.com/mcgibbon/sympl/issues/15]. It might be that we should actually just use .values, would be useful to get your comments in that issue.